april_tag_updates/docking_script.py

```diff
@@ -0,0 +1,132 @@
+expected_dist = 37 #inches
+known_width = 4.18 ##inches
+focal_length = 1.37478492e+03
+from electrical.motor_controller import BasicMotorController
+import numpy as np
+import cv2
+import time 
+from enum import Enum
+
+arucoDict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_APRILTAG_36h11)
+arucoParams = cv2.aruco.DetectorParameters_create()
+cap = cv2.VideoCapture(0)
+window = 'Camera'
+cv2.namedWindow(window)
+def distance_to_camera(knownWidth, focalLength, perWidth):
+  # compute and return the distance from the maker to the camera
+  return (knownWidth * focalLength) / perWidth
+
+def direction_of_tag(corner,image):
+  (_, _, _, _) = corner
+  _,w = image.shape[0], image.shape[1]
+  cx, _ = int(960),int(540)
+  if  bottomLeft[0] > cx:
+    return "right"
+  else:
+    return "left"
+
+idDict = {1: "R1", 2: "R2", 3: "R3"}#TODO add all the other tags
+visited = []
+timeout =  10
+angle_min = 10
+depth_min = 8
```

Comment:
unit of this?

Reply:
visited is a list of tags that we visited already, timeout is in seconds, angle_min is in degrees, and depth_min is in inches.